### DISCUSSION TITLE
Adding javascript in RawHtml Part

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#16398] [Q&A] [answered]
Hi,

I'm trying to add a `<script>` using the HtmlBodypart or the RawHtml flow part, but I notice it gets removed?

How do I include scripts using the content builder?

**Comment by weirdyang** [comment]
Edit the part, uncheck sanitise html.

**Comment by Piedone** [comment]
While yes, you can disable sanitization, I don't recommend adding `<script>` tags into HTML bodies still. Instead, I recommend using Liquid and its tag helpers to include scripts if you need that. That way, the `<script>` won't be in the middle of `<body>`.

**Reply by weirdyang** [reply]
How do I do that?

**Reply by Piedone** [reply]
Add Liquid Part instead of Html Body Part.

**Reply by weirdyang** [reply]
never mind figured it out, I guess u meant use the zone tag helpers

**Comment by Piedone** [comment]
No, the `script` one: https://docs.orchardcore.net/en/latest/reference/modules/Liquid/#script

**Reply by weirdyang** [reply]
but doesn't that mean I need to include that script file first? I want to do some custom JavaScript and put it within script tags

**Reply by Piedone** [reply, marked as answer]
Yes, that's for external script. For an inline one you can use `scriptblock`: https://docs.orchardcore.net/en/latest/reference/modules/Resources/#custom-scripts

**Reply by weirdyang** [reply]
hmm thanks, can't seem to get it to work, will play around with it.
